django/core/serializers/python.py

```diff
@@ -91,6 +92,7 @@ def Deserializer(object_list, **options):
     """
     db = options.pop('using', DEFAULT_DB_ALIAS)
     ignore = options.pop('ignorenonexistent', False)
+    handle_forward_references = options.pop('handle_forward_references', False)
```

Comment:
To prevent incomplete objects, is there a way to detect the missing forward references and then raise an exception that would cause loaddata to rollback the transaction?